frontend/selectors/index.js

```diff
@@ -1,12 +1,83 @@
 import { createSelector } from 'reselect';
+import { getCartProducts } from '@shopgate/pwa-common-commerce/cart/selectors';
+import { getProducts } from '@shopgate/pwa-common-commerce/product/selectors/product';
+import getConfig from '../helpers/getConfig';
+
+const { checkboxValues } = getConfig();
+
+/**
+ * @param {Object} state state
+ * @returns {Object}
+ */
+const getExtensionsState = state => state.extensions;
+
+const getExtensionState = createSelector(
+  getExtensionsState,
+  extensionsState => extensionsState['@shopgate/terms-and-conditions/isOrderableByTerms']
```

Comment:
I think you should set a constant for @shopgate/terms-and-conditions/isOrderableByTerms

Reply:
updated